README.md

```diff
@@ -1719,6 +1720,31 @@ Converts value to a finite number.
 
 **[⬆ back to top](#quick-links)**
 
+### _.isInteger
+
+Checks if value is an integer.
+
+  ```js
+  // Lodash
+  console.log(_.isInteger(3))
```

Comment:
`line 1729,1731` and `line 1735,1737` is duplicated. It is better to show the output for string and integer in both case

Reply:
@cylim thank you. refactored.